docs/tutorials/prowler-app-rbac.md

```diff
@@ -0,0 +1,192 @@
+
+#### Inviting Users
+
+???+note
+    _Please be aware that at this time, an email address can only be associated with a single Prowler account._
+
+Follow these steps to invite a user to your account:
+
+1. Go to [http://localhost:3000/invitations](http://localhost:3000/invitations) from the side menu.
+2. Click on the **Invite User** button on the top right-hand corner of the screen.
+
+    <img src="../img/rbac/invite.png" alt="Invite User" width="700"/>
+
+3. In the Invite User screen, enter the email address of the user you want to invite.
+4. Pick a Role for the user. You can also change the roles for users and pending invites later. To learn more about the roles and what they can do, see [Roles](#roles).
+
+    <img src="../img/rbac/invitation_info.png" alt="Invitation info" width="700"/>
+
+5. Click on the **Send Invitation** button to send the invitation to the user.
+6. After clicking you will see a summary of the status of the invitation. You could access this view again from the invitation menu.
+
+    <img src="../img/rbac/invitation_details.png" alt="Invitation details" width="700"/>
+    <img src="../img/rbac/invitation_details_1.png" alt="Invitation button" width="700"/>
+
+7. To allow the user to join your Prowler account you will need to share the link with the user. They will only need to access this URL and follow the steps to create a user and complete their registration. **Note: Invitations will expire after 7 days.**
+
+    <img src="../img/rbac/invitation_sign-up.png" alt="Invitation sign-up" width="700"/>
+
+???+note
+    If you are a [Prowler Cloud](https://cloud.prowler.com/sign-in) user, the invited user will receive an email with the link to accept the invitation.
+
+#### Editing Invitation
+
+Follow these steps to edit an invitation:
+
+1. Go to [http://localhost:3000/invitations](http://localhost:3000/invitations) from the side menu.
+2. Click on the edit button of the invitation and modify the email, the role or both. **Note: Editing an invitation will not reset its expiration time.**
+
+    <img src="../img/rbac/invitation_edit.png" alt="Invitation edit" width="700"/>
+    <img src="../img/rbac/invitation_edit_1.png" alt="Invitation edit details" width="700"/>
+
+#### Cancelling Invitation
+
+Follow these steps to cancel an invitation:
+
+1. Go to [http://localhost:3000/invitations](http://localhost:3000/invitations) from the side menu.
+2. Click on the revoke button of the invitation.
+
+    <img src="../img/rbac/invitation_revoke.png" alt="Invitation revoke" width="700"/>
+
+#### Sending Invitation Again
+
+To resend the invitation to the user it is necessary to explicitly **delete the previous invitation and create a new invitation**.
+
+## Managing Groups and Roles
+
+The Roles interface in Prowler is designed to facilitate the assignment of custom user privileges. This interface allows administrators to define roles with specific permissions for Prowler administrative tasks and Account visibility.
```

Comment:
```suggestion
The Roles interface in Prowler is designed to facilitate the assignment of custom user privileges. This interface allows administrators to define roles with specific permissions for Prowler administrative tasks and Account visibility.
```

Could you review this line? I don't get the meaning, `interface` seems something technical for a user guide.